textfrag/src/types/directive/fragdirective.rs

```diff
@@ -0,0 +1,336 @@
+/// Fragment Directive object is collection of text fragments in the URL's fragment
+/// :~: is the fragment directive delmiter, followed by list of `[TextDirective]`
```

Comment:
You can use `//!` to make this comment the official module description:

```suggestion
//! Fragment Directive object is collection of text fragments in the URL's fragment
//! :~: is the fragment directive delmiter, followed by list of `[TextDirective]`
```

Reply:
changes done!